packages/gguf/src/gguf.ts

```diff
@@ -1,6 +1,7 @@
 import type { MetadataValue, Version, GGUFMetadata, GGUFTensorInfo, GGUFParseOutput } from "./types";
 import { GGUFValueType } from "./types";
 import { promisesQueue } from "./utils/promisesQueue";
+import { isFrontend } from "../../shared";
```

Comment:
Sorry better copy that file too, I'm deleted `shared` in another PR

Reply:
Resolved in https://github.com/huggingface/huggingface.js/pull/656/commits/1caedf34106013a2e1f7527d6a1db890da6eb2d6